### ISSUE TITLE
bash_completion installation requires root access

### ISSUE THREAD

**Issue by zachriggle** [Gallopsled/pwntools#448]
```
pip install --upgrade git+https://github.com/Gallopsled/pwntools#egg=pwntools
...
    running install_data
    
    copying LICENSE-pwntools.txt -> /home/user/.pyenv/versions/2.7.9/lib/python2.7/site-packages/
    
    copying extra/bash_completion.d/shellcraft -> /etc/bash_completion.d
    
    error: [Errno 13] Permission denied: '/etc/bash_completion.d/shellcraft'
    
    ----------------------------------------
    Rolling back uninstall of pwntools
    Command "/home/user/.pyenv/versions/2.7.9/bin/python2.7 -c "import setuptools, tokenize;__file__='/tmp/user/pip-build-clgHY1/pwntools/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/user/pip-BrsIvq-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/user/pip-build-clgHY1/pwntools
```

**Comment by zachriggle** [comment]
Pinging @kokjo on this since the main instructions were just changed to show this method of installation.

**Comment by Idolf** [comment]
@zachriggle, was this fixed by #636?

**Comment by zachriggle** [comment]
The fix is that the completion is no longer installed automatically (7387ed55a5a99de775111f6d021d67e18ea14bf5)
